test/extended/core.sh

```diff
@@ -51,7 +51,6 @@ SKIP_TESTS=(
   DNS                     # Can't depend on kube-dns
   kube-ui                 # Not installed by default
   DaemonRestart           # Experimental mode not enabled yet
```

Comment:
I've looked into the code and it looks like it's testing restarting nodes and if the cluster will react approprietly, so I guess this should be tested.